packages/ciphernode/events/src/enclave_event/e3_requested.rs

```diff
@@ -17,8 +17,8 @@ impl Display for E3Requested {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         write!(
             f,
-            "e3_id: {}, threshold_m: {}, src_chain_id: {}, seed: {}, params: <omitted>",
-            self.e3_id, self.threshold_m, self.src_chain_id, self.seed
+            "e3_id: {}, threshold_m: {}, src_chain_id: {}, seed: {}, params: {:?}",
+            self.e3_id, self.threshold_m, self.src_chain_id, self.seed, self.params
```

Comment:
Just fyi this was deliberate and done in order to avoid spamming the console with huge data blobs but it might be good to work out a better way to do it.